drivers/modem/gsm_ppp.c

```diff
@@ -709,6 +724,9 @@ static const struct setup_cmd setup_modem_info_cmds[] = {
 	SETUP_CMD("AT+CIMI", "", on_cmd_atcmdinfo_imsi, 0U, ""),
 	SETUP_CMD("AT+CCID", "", on_cmd_atcmdinfo_iccid, 0U, ""),
 #endif
+#if defined (CONFIG_MODEM_FIRMWARE_VERSION)
+	SETUP_CMD("AT+QGMR", "", on_cmd_atcmdinfo_firmware, 0U, ""),
```

Comment:
Note that this is probably a quectel specific command, so maybe check for MODEM_QUECTEL as well.

Reply:
Also check other `#if defined (CONFIG_MODEM_FIRMWARE_VERSION)`

Reply:
Fixed by making `MODEM_FIRMWARE_VERSION` depended on `MODEM_GSM_QUECTEL` in the Kconfig file.